KS.Fiks.Maskinporten.Client/Jwt/JwtRequestTokenGenerator.cs

```diff
@@ -16,7 +17,7 @@ public class JwtRequestTokenGenerator : IJwtRequestTokenGenerator
         private readonly JwtEncoder encoder;
         private readonly X509Certificate2 certificate;
 
-        private static IDictionary<string, object> CreateJwtPayload(string scope, MaskinportenClientConfiguration configuration)
+        private static IDictionary<string, object> CreateJwtPayload(string scope, MaskinportenClientConfiguration configuration, TokenRequest tokenRequest)
```

Comment:
Både scope og consumerOrg kommer fra tokenRequest. Derfor hadde det nok vært bedre her å enten fjerne scope parameter og bruke request eller å legge til et nullable consumerOrg parameter i stedet for tokenRequest

Reply:
Ah, godt poeng.
Jeg endrer så bare TokenRequest sendes inn, så kallet blir litt mer ryddig